utils/opensc/Makefile

```diff
@@ -137,14 +143,14 @@ PROFILES:= \
 	iasecc_generic_pki \
 	iasecc \
 	incrypto34 \
-	jcop \
-	miocos \
```

Comment:
This looks like a regression, just imagine, that you're a `jcop` or `miocos` user and upgrading to this would basically breaks your use cases, right?

Reply:
Is this support still needed?
On openwrt 23.05, the support of jcop and miocos was drop.
Should we still concider this support on 22.03 branch ?
If yes, a patch on 0.24.0 could add this support. what do you think on this solution?

Reply:
> Is this support still needed?

This is a stable release branch, so the goal is to keep it stable, thus the changes should be minimal, ideally keep it as it is and apply only security (or eventually other bug) fixes. Removing features or adding new features should be avoided, there is always a risk of breaking some existing use cases.

Simply look at this from user's perspective, imagine that you've `jcop` or `miocos` tools used somewhere in your project/device which is based on this stable/maintained release, then you notice this update with the security fixes so as a responsible sysadmin you're going to update your devices using `opkg update && opkg upgrade opensc` , but then suddenly realize, that it breaks your use case. Not a great experience, right?

>  On openwrt 23.05, the support of jcop and miocos was drop.

That is OK, its a different stable release branch, so some hiccups are expected during upgrade process between releases.